src/Signer.gs

```diff
@@ -170,6 +174,13 @@
   OAuth.prototype.getSigningKey = function(token_secret) {
     token_secret = token_secret || '';
 
+    // Don't percent encode the signing key (PKCS#8 PEM private key) when using
```

Comment:
This comment could be 2 lines instead of 3.

Reply:
Not if I want to stay within 80 chars. This file doesn't stay within 80 chars (it's 3rd party) but the rest of the library does.